tabs/tab_logit.py

```
@@ -716,9 +721,53 @@ def _run_poisson():
+            plot_html = ""
+            if fig_adj:
+                plot_html = fig_adj.to_html(full_html=False, include_plotlyjs='cdn')
+                poisson_fragment_html += f"<div class='forest-plot-section' style='margin-top: 30px; padding: 10px; border-top: 2px solid #eee;'><h3>🌲 Adjusted Forest Plot</h3>{plot_html}</div>"
+            elif fig_crude:
+                plot_html = fig_crude.to_html(full_html=False, include_plotlyjs='cdn')
+                poisson_fragment_html += f"<div class='forest-plot-section' style='margin-top: 30px; padding: 10px; border-top: 2px solid #eee;'><h3>🌲 Crude Forest Plot</h3>{plot_html}</div>"
+
+            # Wrap in standard HTML structure for standalone download correctness
+            # INLINE CSS FIX: Read static/styles.css and embed directly so downloads look right
+            try:
+                with open('static/styles.css', 'r', encoding='utf-8') as f:
```

Ruff (0.14.10)

Unnecessary mode argument

Remove mode argument

(UP015)